sw/device/lib/dif/autogen/dif_pinmux_autogen.c

```diff
@@ -0,0 +1,9 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+// This file is auto-generated.
+
+#include "sw/device/lib/dif/dif_pinmux.h"
+
+#include "pinmux_regs.h"  // Generated.
```

Comment:
nit: not critical at all but maybe we could skip generation of empty sources. but maybe it will easier to maintain if we just stick with a single pattern.